packages/apollo-gateway/src/__tests__/integration/networkRequests.test.ts

```diff
@@ -0,0 +1,95 @@
+import nock from 'nock';
+import { ApolloGateway } from '../..';
+import createSHA from '../../utilities/createSHA';
+
+import {
+  mockGetRawPartialSchema,
+  mockFetchStorageSecret,
+  mockGetCompositionConfigLink,
+  mockGetCompositionConfigs,
+  mockGetImplementingServices,
+  mockLocalhostSDLQuery,
+} from './nockMocks';
+
+it('Queries remote endpoints for their SDLs', async () => {
+  let url = 'http://localhost:4001';
+  let sdl = `
+  extend type Query {
+      me: User
+      everyone: [User]
+  }
+
+  "My User."
+  type User @key(fields: "id") {
+    id: ID!
+    name: String
+    username: String
+  }
+  `;
+
+  mockLocalhostSDLQuery({ url, sdl });
+
+  let gateway = new ApolloGateway({
```

Comment:
Why not use `const`s for all of these variables which are never mutated?

Reply:
Something to do with three chars vs four.... 🙃. Updated.
For future, there are some good tslint configurations we could be using to handle this sort of thing automatically.